+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Mon Dec  5 11:38:28 2022 - INFO  - Start to run test.
+ expect
spawn sqlite3 ./test.db
couldn't execute "sqlite3": no such file or directory
    while executing
"spawn sqlite3 ./test.db"
++ wc -l ./output.txt
++ grep -cE 25
wc: ./output.txt: No such file or directory
+ CHECK_RESULT 0 1
+ actual_result=0
+ expect_result=1
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 1x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_sqlite_command_08.sh line 48'
+ message='oe_test_sqlite_command_08.sh line 48'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_sqlite_command_08.sh line 48'
Mon Dec  5 11:38:30 2022 - ERROR - oe_test_sqlite_command_08.sh line 48
+ return 0
++ grep -cE select ./output.txt
grep: ./output.txt: No such file or directory
+ CHECK_RESULT '' 1
+ actual_result=
+ expect_result=1
+ mode=0
+ error_log=
+ '[' -z '' ']'
+ LOG_ERROR 'Missing actual error code.'
+ message='Missing actual error code.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Missing actual error code.'
Mon Dec  5 11:38:31 2022 - ERROR - Missing actual error code.
+ return 1
++ wc -l ./output1.txt
++ grep -cE 24
wc: ./output1.txt: No such file or directory
+ CHECK_RESULT 0 1
+ actual_result=0
+ expect_result=1
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 1x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_sqlite_command_08.sh line 50'
+ message='oe_test_sqlite_command_08.sh line 50'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_sqlite_command_08.sh line 50'
Mon Dec  5 11:38:31 2022 - ERROR - oe_test_sqlite_command_08.sh line 50
+ return 0
++ grep select ./output1.txt
grep: ./output1.txt: No such file or directory
+ CHECK_RESULT '' 1
+ actual_result=
+ expect_result=1
+ mode=0
+ error_log=
+ '[' -z '' ']'
+ LOG_ERROR 'Missing actual error code.'
+ message='Missing actual error code.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Missing actual error code.'
Mon Dec  5 11:38:32 2022 - ERROR - Missing actual error code.
+ return 1
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Mon Dec  5 11:38:33 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Mon Dec  5 11:38:34 2022 - ERROR - failed to execute the case.
+ exit 2
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Mon Dec  5 11:38:35 2022 - INFO  - Start to restore the test environment.
+ rm -rf ./test.db './output*.txt'
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Mon Dec  5 11:38:35 2022 - INFO  - End to restore the test environment.
